container/test-app/iframe/iframeContainer.html

```diff
@@ -161,15 +168,6 @@ <h3>
         );
       });
 
-      // SET_THIRD_PARTY_COOKIES_REQUEST - called on microfrontend startup
```

Comment:
Why did you remove this part? We decided to drop support for 3rd party cookies in Luigi v3, not in 'main' branch

Reply:
true. reverted.